components/SocialLoginGroup/SocialLoginButtons.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import { func } from 'prop-types';
+import classNames from 'classnames';
+import GoogleLogin from 'react-google-login';
+import FacebookLogin from 'react-facebook-login/dist/facebook-login-render-props';
+import { facebookKey, googleKey } from 'common/config/environment';
+import styles from './SocialLoginGroup.css';
+
+const SocialLoginButtons = ({ onSuccess, onGoogleFailure }) => (
+  <>
+    <GoogleLogin
+      clientId={googleKey}
+      buttonText="Login"
+      onSuccess={onSuccess('google')}
+      onFailure={onGoogleFailure}
+      render={renderProps => (
+        <button
+          type="button"
+          onClick={renderProps.onClick}
+          disabled={renderProps.disabled}
+          className={classNames(styles.loginButton, styles.googleButton)}
+        >
+          Login with Google
+        </button>
+      )}
+    />
+
+    <FacebookLogin
+      appId={facebookKey}
+      callback={onSuccess('facebook')}
+      render={renderProps => (
+        <button
+          type="button"
+          onClick={renderProps.onClick}
+          className={classNames(styles.loginButton, styles.facebookButton)}
+        >
+          Login with Facebook
+        </button>
+      )}
+    />
+  </>
+);
+
+SocialLoginButtons.propTypes = {
+  onSuccess: func.isRequired,
```

Comment:
Lame nit: can you put proptypes above the definition? So far in the repo proptypes are always at the top of the file.